slides/03/03.md

```diff
@@ -195,7 +195,7 @@ section: ProbabilityBasics
 ## Bernoulli Distribution
 The Bernoulli distribution is a distribution over a binary random variable.
 It has a single parameter $φ ∈ [0, 1]$, which specifies the probability of the random
-variable being equal to 1.
+variable is equal to 1.
```

Comment:
During the merge, I realized it was not correct and rewrote it (to avoid using the "being", and use "that" instead, which is more readable, I think), see https://ufal.mff.cuni.cz/~straka/courses/npfl129/2223/slides/?03#33 for the current version.